spack.yaml

```diff
@@ -5,6 +5,8 @@ spack:
   - 'cmake@3.15:'
   - 'googletest +gmock'
   - 'fftw ~mpi precision=double'
+  - 'fftw ~mpi precision=float'
```

Comment:
I am surprised that the two fftw lines work fine together :/

Reply:
Yes this is what we have in https://github.com/gyselax/gyselalibxx/pull/260.

I guess this led to install different `.so` and somehow worked fine with `FindFFTW.cmake` but definitely not what I recommend to do.